docs/configure-artifact-repository.md

```diff
@@ -554,3 +554,12 @@ configuring the default artifact repository described previously.
       command: [sh, -c]
       args: ["cp -r /my-input-artifact /my-output-artifact"]
 ```
+
+## Artifact Streaming
+
+With artifact streaming, artifacts don’t need to be saved to disk first. Artifact streaming is only supported in the following
+artifact drivers: S3 (v3.4+), HTTP (v3.5+), and Artifactory (v3.5+).
+
+Previously, when a user clicks the download artifact button in the UI, the artifact needs to be written to the
+Argo server’s disk first before downloading. If many users are trying to download at the same time, they will take up
```

Comment:
nitpick: "Previously, when a user _would click_ the _button to download an artifact_ in the UI, the artifact _would need_ to be written"...

Reply:
Thanks!